File: URI Online judge/URI 1180.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        int small=0, index=0;
        int arr [] = new int[n];
        for (int i=0; i<n;i++){
            arr[i] = input.nextInt();
            if (arr[i]<small){
                small = arr[i];index=i;}
        }
        System.out.println("Menor valor: " + small);
        System.out.println("Posicao: " + index);
    }
}

File: URI Online judge/URI 1180.cpp
#include<bits/stdc++.h>

using namespace std;

int main() {
    int n, small=0, index=0;
    int arr[n];
    cin >>n>>arr[n];
    for (int i=0;i<n;i++){
//        cin>>arr[i];
        if (arr[i]<small){
            small=arr[i];
            index=i;
        }
    }
    cout <<"Menor valor: "<<small<<endl;
    cout<<"Posicao: " <<index<<endl;
    return 0;
}

